src/backend/distributed/commands/sequence.c

```diff
@@ -317,6 +317,34 @@ PreprocessDropSequenceStmt(Node *node, const char *queryString,
 }
 
 
+/*
+ * SequenceDropStmtObjectAddress returns list of object addresses in the drop sequence
+ * statement.
+ */
+List *
+SequenceDropStmtObjectAddress(Node *stmt, bool missing_ok)
+{
+	DropStmt *dropSeqStmt = castNode(DropStmt, stmt);
+
+	List *objectAddresses = NIL;
+
+	List *deletingSequencesList = dropSeqStmt->objects;
```

Comment:
`deletingSequencesList` can be confusing `droppingSequencesList` might be easier to follow